### ISSUE TITLE
Instalation support

### ISSUE THREAD

**Issue by bijuscaria** [ngrie/rtmpie#26]
I am trying to install on Ubuntu 22.04 (LTS) x64 

Installation of Docker and Docker Compose done 

then below commands are given but not working. Anything more to do? Please Help

mkdir /opt/rtmpie
cd /opt/rtmpie

wget https://raw.githubusercontent.com/ngrie/rtmpie/main/setup.sh
bash setup.sh 

docker-compose pull
docker-compose up -d

**Comment by bijuscaria** [comment]
root@RTMPGUI:/opt/rtmpie# cd /opt/rtmpie
root@RTMPGUI:/opt/rtmpie# docker compose up -d
[+] Running 4/0
 ⠿ Container rtmpie-php-1       Running                                                                        0.0s
 ⠿ Container rtmpie-caddy-1     Running                                                                        0.0s
 ⠿ Container rtmpie-rtmp-1      Running                                                                        0.0s
 ⠿ Container rtmpie-database-1  Running                                                                        0.0s
root@RTMPGUI:/opt/rtmpie#

**Comment by bijuscaria** [comment]
root@RTMPGUI:/# ufw status
Status: active

To                         Action      From
--                         ------      ----
OpenSSH                    ALLOW       Anywhere
80                         ALLOW       Anywhere
443                        ALLOW       Anywhere
8080                       ALLOW       Anywhere
1935                       ALLOW       Anywhere
OpenSSH (v6)               ALLOW       Anywhere (v6)
80 (v6)                    ALLOW       Anywhere (v6)
443 (v6)                   ALLOW       Anywhere (v6)
8080 (v6)                  ALLOW       Anywhere (v6)
1935 (v6)                  ALLOW       Anywhere (v6)

**Comment by bijuscaria** [comment]
Installation completed successfully as I believe, but I am not getting the web GUI please help

**Comment by bijuscaria** [comment]
root@RTMPGUI:/opt/rtmpie# docker container ls
CONTAINER ID   IMAGE                               COMMAND                  CREATED         STATUS         PORTS                                                                                NAMES
bb3c8d29b946   ghcr.io/ngrie/rtmpie/rtmp:latest    "nginx -g 'daemon of…"   3 minutes ago   Up 3 minutes   0.0.0.0:1935->1935/tcp, :::1935->1935/tcp                                            rtmpie-rtmp-1
6ca41f5765dc   ghcr.io/ngrie/rtmpie/php:latest     "docker-entrypoint p…"   3 minutes ago   Up 3 minutes   9000/tcp                                                                             rtmpie-php-1
a55172be56f4   ghcr.io/ngrie/rtmpie/caddy:latest   "caddy run --config …"   3 minutes ago   Up 3 minutes   0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp, 2019/tcp   rtmpie-caddy-1
865172c04ac2   mariadb:10.4                        "docker-entrypoint.s…"   3 minutes ago   Up 3 minutes   3306/tcp                                                                             rtmpie-database-1
root@RTMPGUI:/opt/rtmpie#

**Comment by Pyromma** [comment]
Did you manage to solve the problem? I'm running into this problem as well.

**Comment by bijuscaria** [comment]
> Did you manage to solve the problem? I'm running into this problem as well.

Yes it works with the link for an example  (demo.yourdomain.com)   this you give during the installation ( Answer the questions)

wget https://raw.githubusercontent.com/ngrie/rtmpie/main/setup.sh
bash setup.sh # Answer the questions
